firebase-config/src/main/java/com/google/firebase/remoteconfig/FirebaseRemoteConfig.java

```diff
@@ -517,17 +459,6 @@ public FirebaseRemoteConfigInfo getInfo() {
     return frcMetadata.getInfo();
   }
 
-  /**
-   * Changes the settings for this {@link FirebaseRemoteConfig} instance.
-   *
-   * @param settings The new settings to be applied.
-   * @deprecated Use {@link #setConfigSettingsAsync(FirebaseRemoteConfigSettings)} instead.
-   */
```

Comment:
We could, but that'd be a breaking API change, too.

Reply:
I wonder if, instead of deleting this one, we could change it's return type to Task and deprecate the setConfigSettingsAsync() method. And then at some point delete the "Async" method. wdyt?

Reply:
Would that be more confusing or error prone for developers on this update path? I'd figure many people use the deprecated sync methods and don't check the return type. Might create unexpected behavior to make them async with the existing name.

Reply:
This PR currently removes the `setConfigSettings` method, so anyone using it is going to have to change code. The same would be true if we retained the method and made it async. Ultimately, I think it's relatively minor, since naming is hard :)

Reply:
Yeah, my thought is that requiring the new method with `Async` in the name will prevent users from blindly switching to a method that's now async (when they previously might have fired and forgot, expecting a sync call).